.buildkite/scripts/stack_tests_windows.ps1

```diff
@@ -45,7 +45,7 @@ fixCRLF
 
 withGolang $env:GO_VERSION
 withDocker $env:DOCKER_VERSION
-withDockerCompose $env:DOCKER_COMPOSE_VERSION
+withDockerCompose $env:DOCKER_COMPOSE_VERSION.Substring(1)
```

Comment:
Required to remove `v` from the version string for the chocolotey tool. Currently, this environment variable has this value `v2.24.1`.

Reply:
Interesting, how has it worked in the past?

Reply:
Probably, it was failing in the past too. I realized that even if with those commands failing the execution of the script continued.